qiskit/compiler/dissasembler.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+# pylint: disable=missing-return-type-doc
+
+"""Disassemble function for a qobj into a list of circuits and it's config"""
+
+from qiskit.converters import qobj_to_circuits
```

Comment:
We should also deprecate `qobj_to_circuits`.

Reply:
Sure I'll add that. I'll move move it over here as a private function in the disassembler module and have` qobj_to_circuits()` call that.

Reply:
Deprecated qobj_to_circuits in: https://github.com/Qiskit/qiskit-terra/pull/2137/commits/9072d61399108a358fc0ad6e74bda55b78ac68cb